### PR TITLE
chore: add CIFS backupstore information

### PR DESCRIPTION
**Pull request by derekbit**

### CODE REVIEW

File: content/docs/1.1.3/concepts.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ Longhorn runs the sync command automatically before creating a snapshot.
 
 # 3. Backups and Secondary Storage
 
-A backup is an object in the backupstore, which is an NFS or S3 compatible object store external to the Kubernetes cluster. Backups provide a form of secondary storage so that even if your Kubernetes cluster becomes unavailable, your data can still be retrieved.
+A backup is an object in the backupstore, which is an S3, NFS or CIFS compatible object store external to the Kubernetes cluster. Backups provide a form of secondary storage so that even if your Kubernetes cluster becomes unavailable, your data can still be retrieved.
```
Comment (innobead) [inline content/docs/1.1.3/concepts.md:266]: CIFS is only supported from 1.5, right?

**Review by innobead — CHANGES_REQUESTED** [review]

File: content/docs/1.1.3/concepts.md (hunk 1 above)
Comment (derekbit) [reply content/docs/1.1.3/concepts.md:266]: Ah. You're right!

Comment (derekbit) [reply content/docs/1.1.3/concepts.md]: Fixed. Thank you.

**Review by innobead — APPROVED** [review]

**Comment by innobead** [comment]
@mantissahz You can also add the azure blob store here since it's also newly added from 1.5.0.

**Pull request merged**